cypress-tests/cypress/support/commands.js

```diff
@@ -151,17 +152,22 @@ Cypress.Commands.add("createPaymentIntentTest", (request, det, authentication_ty
       Accept: "application/json",
       "api-key": globalState.get("apiKey"),
     },
+    failOnStatusCode: false,
     body: request,
   }).then((response) => {
     logRequestId(response.headers['x-request-id']);
     console.log(response);
+
+    expect(res_data.status).to.equal(response.status);
     expect(response.headers["content-type"]).to.include("application/json");
     expect(response.body).to.have.property("client_secret");
     const clientSecret = response.body.client_secret;
     globalState.set("clientSecret", clientSecret);
     globalState.set("paymentID", response.body.payment_id);
     cy.log(clientSecret);
-    expect("requires_payment_method").to.equal(response.body.status);
+    for(const key in res_data.body) {
+      expect(res_data.body[key]).to.equal(response.body[key]);
+    }
     expect(request.amount).to.equal(response.body.amount);
     expect(null).to.equal(response.body.amount_received);
     expect(request.amount).to.equal(response.body.amount_capturable);
```

Comment:
@Sakilmostak aren't we expecting all assertions from connector config?

Reply:
as we discussed, revert this change, we have to combine all such common assertions into one file and for every test case, we should  execute these assertions